bin/wait_for_opensearch

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+timeout="10s"
+status="yellow"
+url="'${OPENSEARCH_HOST}/_cluster/health?wait_for_status=${status}&timeout=${timeout}'"
```

Comment:
We can use opensearch's built-in functionality for this: https://opensearch.org/docs/latest/api-reference/cluster-api/cluster-health/